src/neo4j_genai/core/stores.py

```diff
@@ -0,0 +1,82 @@
+#  Copyright (c) "Neo4j"
```

Comment:
Is there a reason why stores.py and pipeline_graph.py are under /core/? If they are only used in pipelines it may premature optimization to have them in a separate folder

Reply:
No particular reason, code moved in https://github.com/neo4j/neo4j-genai-python/pull/81/commits/5666a4499d092ea29a2f5b7323472d8ebd30d15e